lib/ansible/modules/network/aos/aos_login.py

```diff
@@ -114,14 +115,17 @@ def main():
     module = AnsibleModule(
         argument_spec=dict(
             server=dict(required=True),
-            port=dict(default='8888'),
+            port=dict(default='443', type="int"),
```

Comment:
`DOCUMENTATION` block still says default 8888.

We need to be careful when changing `defaults` in code that's already been released.